tests/ops/qubit/test_parametric_ops.py

```diff
@@ -1017,7 +1023,9 @@ def test_pcphase_tf(self, phi, dim, wires):
         mat2 = op.compute_matrix(*op.parameters, **op.hyperparameters)
 
         expected_mat = tf.Variable(
-            np.diag([np.exp(1j * phi) if i < dim else 1.0 for i in range(2**num_wires)])
+            np.diag(
```

Comment:
How this is computed? If computed by hand or it is a trivial solution, might add a comment to clarify.

Reply:
Its trivial, the result is by definition of the operator